frontend/utils/auth.js

```diff
@@ -73,8 +73,17 @@ export async function getAccessToken() {
 /* removed token, effectively logging out */
 export async function removeToken() {
     // TODO revoke
```

Comment:
if this function is done, this comment should be deleted

Reply:
removed!